### PR TITLE
chore(quick-trace): rename to trace navigator

### PR DESCRIPTION
**Pull request by doralchan**
- Renamed quick trace to trace navigator
- Updated screenshots in docs

### CODE REVIEW

File: src/docs/product/issues/issue-details/index.mdx [hunk 1]
```diff
@@ -56,7 +55,7 @@ You can set your own <PlatformLink to="/enriching-events/breadcrumbs/">breadcrum
 
 ## Trace Details
 
-[Tracing](/product/performance/distributed-tracing/) augments your existing error data by capturing interactions between your software systems. In the Trace Details section, you can click the "Search by Trace" button to see all the events that share the same trace ID.
+[Tracing](/product/sentry-basics/tracing/distributed-tracing/) augments your existing error data by capturing interactions between your software systems. In the Trace Details section, you can click the "Search by Trace" button to see all the events that share the same trace ID.
```
Comment (imatwawana) [inline src/docs/product/issues/issue-details/index.mdx:58]:
```suggestion
[Tracing](/product/sentry-basics/tracing/distributed-tracing/) augments your existing error data by capturing interactions between your software systems. In the "Trace Details" section, you can click the "Search by Trace" button to see all the events that share the same trace ID.
```

File: src/docs/product/sentry-basics/tracing/event-detail.mdx [hunk 2]
```diff
@@ -55,7 +54,7 @@ The waterfall view is a split view where the left reflects the transaction's spa
 
 ### Span Details
 
-Clicking on a span row expands the details of that span. From here, you can see all attached properties, such as tags and other field data. With the Trace ID, you'll be able to view all transactions within that given trace. Click "Search by Trace" to view that Discover list. Learn more about [distributed tracing](/product/performance/distributed-tracing/) in our docs.
+Clicking on a span row expands the details of that span. From here, you can see all attached properties, such as tags and other field data. With the Trace ID, you'll be able to view all transactions within that given trace. Click "Search by Trace" to view that Discover list. Learn more about [distributed tracing](/product/sentry-basics/tracing/distributed-tracing/) in our docs.
```
Comment (imatwawana) [inline src/docs/product/sentry-basics/tracing/event-detail.mdx:57]:
```suggestion
Clicking on a span row expands the details of that span. From here, you can see all attached properties, such as tags and other field data. With the trace ID, you'll be able to view all transactions within that given trace. Click "Search by Trace" to view that **Discover** list. Learn more about [distributed tracing](/product/sentry-basics/tracing/distributed-tracing/) in our docs.
```

File: src/docs/product/sentry-basics/tracing/trace-view.mdx [hunk 3]
```diff
@@ -6,11 +6,11 @@ redirect_from:
...
 ---
 
-With the [trace](/product/performance/distributed-tracing) view, you can drill down into the details of a single trace and traverse every transaction in that trace. This will accelerate your ability to debug slow services, identify related errors, and root out other bottlenecks. There are multiple entry points to get to the trace view.
+With the [trace](/product/sentry-basics/tracing/distributed-tracing) view, you can drill down into the details of a single trace and traverse every transaction in that trace. This will accelerate your ability to debug slow services, identify related errors, and root out other bottlenecks. There are multiple entry points to get to the trace view.
 
 ## Entry Points
-- [Issue Details](/product/issues/issue-details/): In the issue detail, click "View Full Trace" below the Event ID (paired with Quick Trace).
-- [Event Details](/product/performance/event-detail/): In an event detail, click the trace ID on the top right (paired with Quick Trace).
+- [Issue Details](/product/issues/issue-details/): In the issue detail, click "View Full Trace" below the Event ID (paired with Trace Navigator).
```
Comment (imatwawana) [inline src/docs/product/sentry-basics/tracing/trace-view.mdx:12]:
```suggestion
- [Issue Details](/product/issues/issue-details/): In the issue detail, click "View Full Trace" below the event ID (paired with Trace Navigator).
```

File: src/docs/product/sentry-basics/tracing/trace-view.mdx [hunk 4]
```diff
@@ -30,7 +30,7 @@ The transaction view provides a split view similar to the [span view](/product/s
 
 ### Orphan Traces and Broken Subtraces
 In a fully instrumented system, a span in one service will connect to a transaction in a subsequent service. For a variety of reasons, a transaction in a service may not arrive in Sentry. When Sentry encounters these types of transactions within a trace, we link the transactions with a dashed line since they can no longer be directly connected to the root.
-In addition, broken subtraces can occur when Sentry receives an error for a trace, but the corresponding transaction does not exist. In this case, the error will not appear on the trace view and the quick trace won't appear on the Issue Details page. In these cases you can click "Open In Discover" to see all the events in the trace.
+In addition, broken subtraces can occur when Sentry receives an error for a trace, but the corresponding transaction does not exist. In this case, the error will not appear on the trace view and the trace navigator won't appear on the Issue Details page. In these cases you can click "Open In Discover" to see all the events in the trace.
```
Comment (imatwawana) [inline src/docs/product/sentry-basics/tracing/trace-view.mdx:33]:
```suggestion
In addition, broken subtraces can occur when Sentry receives an error for a trace, but the corresponding transaction does not exist. In this case, the error will not appear on the trace view and the trace navigator won't appear on the **Issue Details** page. In these cases you can click "Open In Discover" to see all the events in the trace.
```

**Review by imatwawana — APPROVED** [review]
Minor style edits

**Review by dashed — APPROVED** [review]

**Pull request merged**